### PR TITLE
Update Documentation for Logics

### PR DESCRIPTION
**Pull request by Seris370**

### CODE REVIEW

**Comment by Seris370** [comment]
fix #82

File: src/main/java/seedu/algobase/model/AlgoBase.java [hunk 1]
```diff
@@ -176,6 +176,7 @@ public void removeTag(Tag key) {
 
     /**
      * Replaces the contents of the Plan list with {@code plans}.
+     * {@code plans} must not contain duplicate plans.
```
Comment (jiayushe) [inline src/main/java/seedu/algobase/model/AlgoBase.java:179]: I think we don't check for duplicate plans?

**Review by jiayushe — APPROVED** [review]
LGTM. Just remove the line in header.

File: src/main/java/seedu/algobase/model/AlgoBase.java (hunk 1 above)
Comment (Seris370) [reply src/main/java/seedu/algobase/model/AlgoBase.java:179]: We don't allow plans with duplicate names for now.

**Pull request merged**